specs/deneb/p2p-interface.md

```diff
@@ -119,11 +120,12 @@ The *type* of the payload of this topic changes to the (modified) `SignedBeaconB
 
 ###### `blob_sidecar_{index}`
 
-This topic is used to propagate signed blob sidecars, one for each sidecar index. The number of indices is defined by `MAX_BLOBS_PER_BLOCK`.
+This topic is used to propagate signed blob sidecars, one for each sidecar index. The number of indices is defined by `MAX_BLOBS_PER_BLOCK` which can be upgraded on via a fork. However the actual blobs that might be generated by the EL will be limited by `MAX_DATA_GAS_PER_BLOCK // DATA_GAS_PER_BLOB` (requires an EL hardfork to upgrade the data gas limit) but should never exceed `MAX_BLOBS_PER_BLOCK` limit of the corresponding CL fork.
 
 The following validations MUST pass before forwarding the `signed_blob_sidecar` on the network, assuming the alias `sidecar = signed_blob_sidecar.message`:
 
 - _[REJECT]_ The sidecar is for the correct topic -- i.e. `sidecar.index` matches the topic `{index}`.
+- _[REJECT]_ The `sidecar.index` is lower than `MAX_BLOBS_PER_BLOCK`.
```

Comment:
this seems like something we should check outside the networking stack, e.g only configure N subnets and then we get this check by the check defined right before it that the topic matches the index